cpp/modmesh/simd/neon/neon.hpp

```diff
@@ -222,6 +224,267 @@ void div(T * dest, T const * dest_end, T const * src1, T const * src2)
     }
 }
 
+/**
+ * @brief 使用SIMD加速的三数取中法选择pivot
```

Comment:
Everything in the code base, including comments, should should use English.

The comments do not look like your words.  If you copied the code from elsewhere, please close this PR and avoid doing that.  I do not want this repository to use code copied from elsewhere.

If you generated the code using AI, it is not right to directly take the comments from the generated code.  We should inspect the generated code line by line and fully understand it, including comments.

Any code generated from AI (or copied from elsewhere) should be explicitly indicated in PR inline annotations.

The comments in the code are over-explaining.  It suggests the code does not speak for itself, and should be improved.